core/src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala

```diff
@@ -331,7 +331,7 @@ class RemoteServicesMockServer(port:Int) extends RawlsTestUtils {
       response()
         .withHeaders(jsonHeader)
         .withBody {
-          (1 to numWorkflows).map(_ => ExecutionServiceStatus(UUID.randomUUID().toString, "Submitted")).toList.toJson.toString
+          (1 to numWorkflows).map(n => ExecutionServiceStatus(UUID.randomUUID().toString, if (n == numWorkflows) "Failed" else "Submitted")).toList.toJson.toString
```

Comment:
Yeah. 

BTW this is one of the reasons I like Mockito: it makes every test responsible for its own mocked data so you're not guessing which test uses which piece of a global test data set.

http://www.scalacheck.org/ is another nice tool for test data generation & property based testing -- I could give the team an overview sometime if interested.